PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingDetailVC.swift

```diff
@@ -267,7 +311,26 @@ class PetListingDetailVC: UITableViewController
         return UITableViewCell()
     }
 
-    ////////////////////////////////////////////////////////////
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+
+        let sectionNumber = indexPath.section
+        let rowNumber = indexPath.row
+        
+        if sectionNumber == 3 && rowNumber == 1 {
+            let mailComposeViewController = configureMailController()
+            if MFMailComposeViewController.canSendMail() {
+                self.present((mailComposeViewController)!, animated: true, completion: nil)
+            } else {
+                displayEmailError()
+            }
+        }
+
+//        If the return type for configureMailController() is optional like I suggested above,
+//        you can wrap all of this code in an if let and only try to
+//        present the controller if there is actually an email address.
+//
```

Comment:
These look like the comments I made in my last review hehe...  You can go ahead and remove these.

Reply:
Removed it!